templates/plugins/selectfield.html

```diff
@@ -1,27 +1,80 @@
 {% load custom_filters %}
 
 <style>
-    .select2-container{
-        padding:0px;
+    .select2-container {
+        padding: 0px;
     }
 </style>
-<select class="col-md-12 {{attrs.class}}" id="{{name}}" name="{{name}}">
-    <option></option>
-    {% for key,val in options %}
-        <option value="{{key}}" {%if val == value %}selected{%endif%}>{{val}}</option>
-    {%endfor%}
-</select>
+
+<!--data-multiple="{{multiple}}"-->
+<input type="hidden"
+       class="col-md-12 {{attrs.class}}"
+       value="{% if value %}{{ value|safe }}{% endif %}"
+       id="{{name}}"
+       name="{{name}}"
+/>
 
 <script>
-    $(document).ready(function(){
+    $(document).ready(function() {
```

Comment:
delete the unused code in this function and remove the console.logs, other than that, LGTM

Reply:
Done!